ext/dapr-ext-workflow/dapr/ext/workflow/aio/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from .dapr_workflow_client import DaprWorkflowClientAsync
+
+# Public alias to mirror sync naming under aio namespace
+DaprWorkflowClient = DaprWorkflowClientAsync
+
+__all__ = [
+    'DaprWorkflowClientAsync',
+    'DaprWorkflowClient',
```

Comment:
I think it'd be better to have only one exposed. I'd even rename the class to `DaprWorkflowClient` (dropping the `Async`) to match the `from dapr.aio.clients import DaprClient`. What do you think?